sources/tests/xunit.runner.stride/Views/MainWindow.xaml.cs

```diff
@@ -9,9 +9,6 @@ public class MainWindow : Window
         public MainWindow()
         {
             InitializeComponent();
-#if DEBUG
-            this.AttachDevTools();
-#endif
```

Comment:
This has been moved to its own nuget, I don't think it was used so I ended up removing it to avoid including another package.